tests/acceptance/view-user-profile-test.js

```diff
@@ -63,6 +81,11 @@ module('Acceptance | view-user-profile', function (hooks) {
     });
 
     await userPage.visit({ username: 'rohitpaulk' });
+    assert.strictEqual(userPage.courseLabels[0].text, 'Build your own Docker');
+    // Completed course will be listed first.
+    assert.strictEqual(userPage.courseLabels[1].text, 'Build your own grep');
+    assert.strictEqual(userPage.courseLabels[2].text, 'Build your own Redis');
+    // Then ordering is done on latest submission, earlier comes first.
```

Comment:
Same as above - these comments are really confusing, it feels like you're commenting on the `percySnapshot` line below